frontend/cypress/tests/chartView/chartView_toolBar_searchBox.cy.js

```diff
@@ -33,4 +33,100 @@ describe('search bar', () => {
       expect(children).to.equal(1);
     });
   });
+
+  it('searching by non-existent tag shows no results', () => {
+    cy.get('#app #tab-resize .tab-close').click();
+    cy.get('#summary-wrapper input[type=text]')
+      .type('tag: asdfghjkl')
+      .type('{enter}');
+
+    // Enter does not work. Related issue: https://github.com/cypress-io/cypress/issues/3405
+    // Let's manually submit form
+    cy.get('#summary-wrapper form.summary-picker')
+      .submit();
+
+    cy.get('#summary-wrapper #summary-charts')
+      .should('be.empty');
+  });
+
+  it("searching tag that only exists in one author's commits shows one result", () => {
+    cy.get('#app #tab-resize .tab-close').click();
+    cy.get('#summary-wrapper input[type=text]')
+      .type('tag: v1.8')
+      .type('{enter}');
+
+    // Enter does not work. Related issue: https://github.com/cypress-io/cypress/issues/3405
+    // Let's manually submit form
+    cy.get('#summary-wrapper form.summary-picker')
+      .submit();
+
+    cy.get('#summary-wrapper .summary-charts')
+      .find('.summary-chart')
+      .its('length')
+      .should('eq', 1);
+  });
+
+  it("searching tag that only exists in two authors' commits shows two results", () => {
+    cy.get('#app #tab-resize .tab-close').click();
+    cy.get('#summary-wrapper input[type=text]')
+      .type('tag: v1.10')
+      .type('{enter}');
+
+    // Enter does not work. Related issue: https://github.com/cypress-io/cypress/issues/3405
+    // Let's manually submit form
+    cy.get('#summary-wrapper form.summary-picker')
+      .submit();
+
+    cy.get('#summary-wrapper .summary-charts')
+      .find('.summary-chart')
+      .its('length')
+      .should('eq', 2);
+  });
+
+  it("search field doesn't start with 'tag:' prefix but still contains it shows no results", () => {
+    cy.get('#app #tab-resize .tab-close').click();
+    cy.get('#summary-wrapper input[type=text]')
+      .type('v1.10 tag: v1.10')
+      .type('{enter}');
+
+    // Enter does not work. Related issue:
+    // Let's manually submit form
+    cy.get('#summary-wrapper form.summary-picker')
+      .submit();
+
+    cy.get('#summary-wrapper #summary-charts')
+      .should('be.empty');
+  });
+
+  it("search field doesn't contain 'tag:' at all shows no results", () => {
+    cy.get('#app #tab-resize .tab-close').click();
+    cy.get('#summary-wrapper input[type=text]')
+      .type('v1.10')
+      .type('{enter}');
+
+    // Enter does not work. Related issue:
+    // Let's manually submit form
+    cy.get('#summary-wrapper form.summary-picker')
+      .submit();
+
+    cy.get('#summary-wrapper #summary-charts')
+      .should('be.empty');
+  });
+
+  it('searching for multiple tags shows results containing all the tags searched', () => {
```

Comment:
For the tests, would it be possible to also assert that the results contain the tags that are searched for? If I understand correctly, currently the tests would pass even if the search results don't match the searched tags (but still return the correct expected number of results).

If this is not possible, it might be a good idea to add a comment about the assumptions of each test since we are hardcoding the search values (e.g. Assumption: There are exactly 2 charts that match the tag "bb v1.10".)